requirements.txt

```diff
@@ -18,7 +18,7 @@ filelock
 networkx
 jinja2
 fsspec
-lintrunner
+lintrunner ; platform_system != "Windows" or (platform_system == "Windows" and platform_machine == "x86")
```

Comment:
```suggestion
lintrunner ; platform_system != "Windows"
```

I don't think there is any plan to support it on any windows version unfortunately.

Reply:
@albanD okay, let's skip Windows all version.